tests_v2/test_focus_history_shortcut_cross_workspace.py

```
@@ -0,0 +1,102 @@
+    # Closed workspaces are skipped, matching the arrow buttons' pruning.
+    c.close_workspace(created[1])
+    time.sleep(0.3)
+    got = _press_and_wait(c, "cmd+[", created[0])
+    _must(got == created[0], f"cmd+[ should skip closed ws2 and land on ws1, got {got}")
+
+    # Cleanup the workspaces this test created.
+    for wsid in (created[0], created[2]):
+        try:
+            c.close_workspace(wsid)
+        except cmuxError:
+            pass
```

Ruff (0.16.0)

[warning] Use `contextlib.suppress(cmuxError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(cmuxError): ...`

(SIM105)